packages/orbit-components/package.json

```diff
@@ -73,8 +73,8 @@
   },
   "peerDependencies": {
     "@babel/runtime": "^7.21.0",
-    "react": ">=19.1.0",
-    "react-dom": ">=19.1.0"
+    "react": ">=19.0.0",
```

Comment:
Why is there `>=` and not `^`? I see the version was `greater than or equal` before, is there any reason why we don't want to set the caret char here (and also for `react-dom` peer dep)?

Reply:
Yes, the Booking example is a good one. We allow for further majors. They usually support older versions as well. So we don't block teams from being on the latest versions.

Reply:
ok, that makes sense in that case. 👍